samtranslator/model/stepfunctions/events.py

```diff
@@ -259,6 +256,7 @@ def resources_to_link(self, resources):
         permitted_stage = "*"
         stage_suffix = "AllStages"
         explicit_api = None
+        rest_api_id = PushApi.get_rest_api_id_string(self.RestApiId)
```

Comment:
Not sure if it's weird calling a push api method from a state machine api method